src/utilities/index.ts

```diff
@@ -86,3 +86,33 @@ export const noScroll = {
     document.querySelector('html')!.classList.toggle('no-scroll')
   },
 }
+
+export interface IndexSignature {
+  [key: string]: any
+}
+
+/**
+ * Pick out matching keys of object
+ *
+ * @param obj - object of properties
+ * @param keys - array of keys to pick out of the object
+ * @returns {object} - an object of matching keys only
+ */
+export function pick(obj: IndexSignature, keys: string[]): IndexSignature {
+  return keys
+    .map(k => (k in obj ? { [k]: obj[k] } : {}))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline between consequent and alternate of ternary expression.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)